charts/k8ssandra-operator/Chart.yaml

```diff
@@ -3,14 +3,14 @@ name: k8ssandra-operator
 description: |
   Kubernetes operator which handles the provisioning and management of K8ssandra clusters.
 type: application
-version: 0.38.4
+version: 0.38.5
```

Comment:
You'll need to rebase and increment the version again as main is already on `0.38.5` for the `1.4.0` release.